### DISCUSSION TITLE
Notifications for new API Releases

### DISCUSSION THREAD

**Discussion by albusse** [Azure/api-management-developer-portal#2541] [Feature request]
Not quite sure, if i'm right here but i will just try to ask ;)

Is there a way to inform users about new API releases? We do have the changelog tab but is there a way to add sth like the "Watch"-Button on GitHub?

**Comment by azaslonov** [comment]
Hi @albusse, do you need something besides this one?

![image](https://user-images.githubusercontent.com/2320302/79590517-feb05580-808b-11ea-8c5a-eb98dd61fe38.png)

**Comment by albusse** [comment]
Something like this but I mean for the managed azure developer portal.
When we as a company release a new api version or revision we would like to automatically inform the willing customer developers Based on the changelog of the new API Revision.

**Comment by azaslonov** [comment]
I see. @mikebudzynski do you want to comment on this?

**Comment by mikebudzynski** [comment]
@albusse, apologies for a delay in response and thanks for raising this feature request - it makes a lot of sense.

1. How would you imagine customers being notified? Via email? 
2. Instead of relying on them clicking the "watch" button, would it make sense to notify them about changes to ANY API they have access? What about any API they have used in the past X days?

For the time being, as a mitigation, you can create a workflow (for example, with Azure Logic Apps), which would trigger notifications to your API consumers.

**Comment by abusse-interhyp** [comment]
Thx for your answer @mikebudzynski 
I will give it a try.
For the developer portal, email should be a Good way.
With your suggestion maybe on product level. We can have a opt-in functionality with a text via registration or subscription of a product. But we would at least need an opt-out button for this mails.
Maybe a product based widget with last changes (based on api changelogs) could also be a good first alternative?
